2. LLP distribution.wls

```diff
@@ -148,19 +168,24 @@
 (*{mnucleus["Mo"],anucleus["Mo"],znucleus["Mo"]}={89.38,96,42};*)
 (*{mnucleus["Fe"],anucleus["Fe"],znucleus["Fe"]}={54.43,56,28};*)
 (*{mnucleus["C"],anucleus["C"],znucleus["C"]}={11.188,12,6};*)
+(*(*{mnucleus["H"],anucleus["H"],znucleus["H"]}={1.0078,1,1};*)*)
+(**)
 (*(*Br ratios of EM decays of light mesons*)*)
 (*Br\[Pi]To\[Gamma]\[Gamma]=0.98799;*)
 (*Br\[Eta]To\[Gamma]\[Gamma]=0.3931181;*)
 (*(*Photon absorption cross-sections*)*)
 (*{\[Sigma]\[Gamma]Abs["Mo"],\[Sigma]\[Gamma]Abs["Fe"]}={12.7,5.1}*GeVm2Tob^-1;*)
 (*\[Sigma]\[Gamma]Abs["C"]=\[Sigma]\[Gamma]Abs["Fe"]*(znucleus["C"]*(znucleus["C"]+1)Log[183/znucleus["C"]^(1/3.)])/(znucleus["Fe"]*(znucleus["Fe"]+1)Log[183/znucleus["Fe"]^(1/3.)]);*)
+(*\[Sigma]\[Gamma]Abs["H"] = 10^-14*)
+(**)
 (*(*Nucleus-proton cross-section in pb*)*)
-(*Do[\[Sigma]pNucleus[Z]=53*10^9*anucleus[Z]^0.77,{Z,{"Mo","Fe","C"}}];*)
+(*(*Do[\[Sigma]pNucleus[Z]=53*10^9*anucleus[Z]^0.77,{Z,{"Mo","Fe","C"}}];*)*)
+(*Do[sigmaNucleus[Z]=If[Z=="H",42.5*10^9,53*10^9*anucleus[Z]^0.77],{Z,{"Mo","Fe","C","H"}}]*)
 (*(*proton-proton cross-section in pb*)*)
-(*{\[Sigma]pp["Serpukhov"],\[Sigma]pp["FermilabBD"],\[Sigma]pp["SPS"],\[Sigma]pp["LHC"],\[Sigma]pp["FCC-hh"],\[Sigma]pp["ESS"]}={40.,40.,40.,78.,104.,44.}*10^9;*)
+(*{\[Sigma]pp["Serpukhov"],\[Sigma]pp["FermilabBD"],\[Sigma]pp["SPS"],\[Sigma]pp["LHC"],\[Sigma]pp["FCC-hh"],\[Sigma]pp["ESS"], \[Sigma]pp["LHC-fixed-target"] }={40.,40.,40.,78.,104.,44.,42.5}*10^9;*)
```

Comment:
This part you'll have to explain to me.

Reply:
Okay, I'm just defining the cross-section nucleus-proton, fixing a value of 42.5*10^9 for H, and keeping the other values as previously defined. 
But this, and the definition of the  nucleus parameters (m,Z,A), are only relevant for ALP-photon, so they don't affect the results for the other LLPs.